README.md

```diff
@@ -134,7 +134,9 @@ await pc.describeIndex('serverless-index');
 //    dimension: 1536,
 //    metric: 'cosine',
 //    host: 'serverless-index-4zo0ijk.svc.us-west2-aws.pinecone.io',
+//    deletionProtection: 'disabled',
 //    spec: {
+//       pod: undefined,
```

Comment:
You can leave this off completely if it's not needed, and vice versa. No need to add it.